src/hooks/useBalanceChartData/useBalanceChartData.ts

```diff
@@ -366,15 +371,34 @@ export const useBalanceChartData: UseBalanceChartData = args => {
     }
 
     // create empty buckets based on the assets, current balances, and timeframe
-    const emptyBuckets = makeBuckets({ assetIds, balances, timeframe })
+    const emptyBuckets = makeBuckets({
```

Comment:
yeah sorry we can't do this hear, i just cleaned this up recently.

this hook is highly optimized and already complicated enough.

you can achieve the same outcome by putting this logic in the selector layer

Reply:
Synced with @0xdef1cafe and agree to keep this as a *temporary* hack.